lucien-ios/Dashboard/DashboardCollectionView/DashboardCollectionViewCell.swift

```diff
@@ -27,6 +27,13 @@ final class DashboardCollectionViewCell: UICollectionViewCell {
         setImage(imageURL: imageURL, storyTitle: storyTitle)
         setStyling()
     }
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)